datasets/hess-crab4/make_tables.py

```diff
@@ -11,13 +11,16 @@
 rows = []
 for obs in obs_ids:
     for filetype in file_types:
-        name = "hess_" + filetype + "_" + obs + ".fits.gz"
+        #name = "hess_" + filetype + "_" + obs + ".fits.gz"
+        name = "hess_{}_{:06d}.fits.gz".format(filetype, obs)
         if filetype == 'events':
-            name = "hess_events_simulated_" + obs + ".fits"
+        #    name = "hess_events_simulated_" + obs + ".fits"
+            name = "hess_events_simulated_{:06d}.fits".format(obs)
         if filetype == 'background':
-            name = "hess_bkg_offruns_" + obs + ".fits.gz"
+        #    name = "hess_bkg_offruns_" + obs + ".fits.gz"
+            name = "hess_bkg_offruns_{:06d}.fits.gz".format(obs)
         data = dict()
-        data['OBS_ID'] = int(obs[1:])
```

Comment:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/Users/jouvin/Desktop/these/test_Gammapy/gammapy-extra/datasets/hess-crab4/make_tables.py in <module>()
     21             name = "hess_bkg_offruns_{:06d}.fits.gz".format(obs)
     22         data = dict()
---> 23         data['OBS_ID'] = obs[1:]
     24         data['TYPE'] = filetype
     25         data['NAME'] = name

TypeError: 'int' object has no attribute '__getitem__'

In [3]: obs[1:]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-183a113df849> in <module>()
----> 1 obs[1:]

TypeError: 'int' object has no attribute '__getitem__'